### ISSUE TITLE
Missing icon custom connector after import

### ISSUE THREAD

**Issue by KurtStam** [syndesisio/syndesis#4055]
## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[*] Bug report  
[ ] Documentation issue or request
</code></pre>

## Description
The icon should be exported along with the integration, so that on import it reappears

<img width="599" alt="screen shot 2018-11-08 at 11 09 07 am" src="https://user-images.githubusercontent.com/35576/48191951-c230e400-e346-11e8-9600-22615298ea17.png">

Todo API icon is missing on fresh install and re-import.

https://github.com/syndesisio/syndesis-quickstarts/raw/master/db-2-api-connector/DB-2-APIConnector-export.zip

**Comment by zregvart** [comment]
@KurtStam can you take a look at the `src` of that image, should be something like:

```html
<img class="syn-icon-small" src="./../../assets/icons/multi-flow.connection.png" alt="1 Flow">
```

**Comment by KurtStam** [comment]
@zregvart here you go

> <img _ngcontent-c8="" class="syn-icon-medium" src="/api/v1/connectors/i-LQcy-E1cre0V-4vObkwz/icon?db:i-LQcyc_Pcre0V-4vObkxz">

**Comment by zregvart** [comment]
The thing is that the UI replaces icon (see #3974) and I guess then posts it as the integration is being saved. I think this should have been done on the backend when the integration is created so we can reference a predefined icon on the backend. 

Seems to me that what we have has two drawbacks:
 - we get a new icon but identical with each flow stored in the database
 - we need to implement support for exporting action icons in integration export

Since #3974 was a p0, should we consider this a p0 also and rework this?

cc @paoloantinori @gashcrumb

**Comment by KurtStam** [comment]
Ready for QE

**Comment by mastepan** [comment]
Fix verified on 1.6.1-20181203
